docs/Installation/Linux CLI command: pt.md

```diff
@@ -0,0 +1,35 @@
+# Papertrail Linux CLI command: pt
+
+A command line interface to perform functions not found in the administrative GUI is available. This utility can be installed on Linux via pip (package management system to install Python software packages).
```

Comment:
It can also be used as part of a scripting process e.g. bulk export/upload etc..